docs/getting_started/quick_start.md

```diff
@@ -0,0 +1,50 @@
+# Quick Start
+
+Ribir is a cross-platform GUI library for Rust. As a GUI library, Ribir helps you build graphics user interface for your Rust applications.
+
+Now, You can set up a Ribir project easily and quickly. Let's try it out and output "hello world"!
+
+## Setting up Ribir
+
+The first you need create a new Rust Application. 
+
+> Tips
+> 
+> Ribir only support rust nightly version now.
+
+```shell
+cargo new ribir-hello-world
+cd ribir-hello-world
+```
+
```

Comment:
Nightly tips place here better.   And  tell the user can use  `rustup override set nightly` switch channel.